multi-agent/lib/mas/blueprints/service.py

```
@@ -159,6 +172,27 @@ def get_draft_schema() -> Dict[str, Any]:
+            raise PromptShortcutsValidationError(str(exc)) from exc
+        if not self._repo.set_prompt_shortcuts(blueprint_id=blueprint_id, prompts=shortcuts.to_storage()):
+            raise BlueprintNotFoundError(blueprint_id)
+        return shortcuts
+
+    def get_prompt_shortcuts(self, blueprint_id: str) -> PromptShortcuts:
+        if not self.exists(blueprint_id):
+            raise BlueprintNotFoundError(blueprint_id)
+        try:
+            raw = self._repo.get_prompt_shortcuts(blueprint_id=blueprint_id)
+        except KeyError:
+            raise BlueprintNotFoundError(blueprint_id)
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)